### ISSUE TITLE
does not install on Fedora 25

### ISSUE THREAD

**Issue by silberzwiebel** [bryanherger/xdrawchem#7]
Hi,
I tried to use the provided .rpm package to install xdrawchem under fedora 25.
This fails, however:

```
Could not depsolve transaction; 1 problem detected:
0. nothing provides libopenbabel.so.3()(64bit) needed by xdrawchem-1.10.2-1.x86_64
```

Package `openbabel` is installed, fedora has version 2.3.90. It shows up as `/usr/lib64/libopenbabel.so.4.0.0.`
A manual symlink `libopenbabel.so.3 -> libopenbabel.so.4.0.0` didn't work (possibly because the pacakge installer does not look whether the file exists but only if there is a package that provides it, I guess ...)

Is this version too new? Could you update the .rpm for Fedora?

Thanks!

**Comment by bryanherger** [comment]
Hi, I built the RPM on CentOS, which doesn't provide OpenBabel.  I added the EPEL repo to get OpenBabel, which provided:

openbabel-2.2.3-1.el6.i686 : Chemistry software file format converter
Repo        : epel
Matched from:
Filename    : /usr/lib/libopenbabel.so.3

Fedora 25 and later provide OpenBabel without needing EPEL, and they provide the newer 2.4.x build.

So there are two workarounds possible here:
1) Import EPEL repo and downgrade your OpenBabel to that version;
2) Recompile XDrawChem from source so it will link to the newer OpenBabel library.

For my part, the next version will probably link libopenbabel.a statically to avoid version mismatch, unless someone can suggest a smarter option.

**Comment by silberzwiebel** [comment]
Ok, thanks. I found a third workaround, because I neither wanted to compile nor add a whole repo with potential package conflicts all over. So I just uninstalled Fedora's openbabel version and then downloaded [the corresponding openbabel rpm](https://dl.fedoraproject.org/pub/epel/6/x86_64/openbabel-2.2.3-1.el6.x86_64.rpm) from the EPEL repo and installed it. After this, your rpm package installs fine and xdrawchem works.
